### ISSUE TITLE
low map95-50

### ISSUE THREAD

**Issue by ShevchenkoKatarina** [ultralytics/ultralytics#20237]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Low map50-95 around 0.2, for 150-200 epochs when looking for graffiti on panoramas, objects were pre-cut because the resolution is too high around 13000 by 6500. What could be the problem? Screen markup.

![Image](https://github.com/user-attachments/assets/369f8eb2-8211-4a5a-9844-16e8f1633f65)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ShevchenkoKatarina, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://reddit.com/r/Ultralytics) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

---

This is an automated response 🤖. An Ultralytics engineer will assist you here soon!

**Comment by Y-T-G** [comment]
You shouldn't be using objects that are already cropped. Otherwise the model will not learn how to detect objects in uncropped images.

**Comment by ShevchenkoKatarina** [comment]
> You shouldn't be using objects that are already cropped. Otherwise the model will not learn how to detect objects in uncropped images.

If the images are too large, about 13000 by 6500, that is 360 degree panoramas. And the objects themselves are small enough, power doesn't allow me to use imgsz larger than 2000 pixels. With this compression the graffiti is almost indistinguishable in the images.

**Comment by ShevchenkoKatarina** [comment]
> You shouldn't be using objects that are already cropped. Otherwise the model will not learn how to detect objects in uncropped images.

If I train on large images, without cropping it gives me no more than 0.22 mAP50-95

**Comment by ShevchenkoKatarina** [comment]
> You shouldn't be using objects that are already cropped. Otherwise the model will not learn how to detect objects in uncropped images.

In general, after the training, there was an idea to use SAHI, and cut the images to search for graffiti

**Comment by Y-T-G** [comment]
What's your training command?

Can you post the `val_batch_labels.jpg` files from your `runs` folder?
